src/pyMetricCli/adapter_interface.py

```diff
@@ -33,6 +33,8 @@
 # Imports
 ################################################################################
 
+from abc import ABC, abstractmethod
```

Comment:
In __main__.py, you still do all the checks of the given adapter --> do you want to keep the checks?